code/mob/living/intangible/flockmob_parent.dm

```diff
@@ -17,6 +17,10 @@
 	var/datum/flock/flock = null
 	var/wear_id = null // to prevent runtimes from AIs tracking down radio signals
 
+	var/afk_counter = 0
+	var/afk_counter_threshold = 180 SECONDS
```

Comment:
This should probably be a define instead of a var; I don't see a situation in which it would vary for different mobs or change during a round.